src/overview/bare-metal.md

```diff
@@ -1,59 +1,53 @@
-# Bare Metal (`no_std`)
+# Developing on Bare Metal (`no_std`)
 
-Using `no_std` may be more familiar to embedded Rust developers; it does not use `std` (the Rust standard library) but instead uses a subset, the `core` library. The [official Rust embedded book] has a [great section on this].
+Using `no_std` may be more familiar to embedded Rust developers; it does not use `std` (the Rust standard library) but instead uses a subset, the `core` library. [The Embedded Rust Book][embedded-rust-book] has a great [section][embedded-rust-book-no-std] on this.
 
 It's important to note that in general a `no_std` crate can always compile in `std` environment but the inverse is not true. Therefore, when creating crates it's worth keeping in mind if it needs the standard library to function.
 
-[great section on this]: https://docs.rust-embedded.org/book/intro/no-std.html
-[official rust embedded book]: https://docs.rust-embedded.org/
+[embedded-rust-book]: https://docs.rust-embedded.org/
+[embedded-rust-book-no-std]: https://docs.rust-embedded.org/book/intro/no-std.html
 
-## Hardware Abstraction Layers
 
-Previously, the primary focus of `no_std` development was the ESP32 and (to a lesser extent) the ESP8266.
+## Relevant `esp-rs` crates
 
-Now there is a renewed effort to implement `no_std` support for the entire lineup of Espressif devices from the ESP32 and newer. These new HALs can be found in the [esp-rs/esp-hal] repository.
+The table below covers what `no_std` features are supported by Espressif products.
 
-There is also some level of support for Wi-Fi and Bluetooth via [esp-rs/esp-wifi] for ESP32, ESP32-C3, ESP32-S2, and ESP32-S3.
+> **Notes**:
+>
+> - Hover your cursor over the links in the table header to see feature details.
```

Comment:
```suggestion
```

I would remove this comment, in my opinion, seeing a link on it should be self-explanatory, and we also have an explanation on the list bellow

Reply:
Removed